invalidate-cloudfront.js

```diff
@@ -0,0 +1,49 @@
+const exec = require('child_process').exec;
+const colors = require('colors');
+const AWS = require('aws-sdk');
+
+const cloudfront = new AWS.CloudFront();
+const pathToInvalidate = `/react-vapor/*`;
+
+const shouldDoInvalidation = () => {
+    if (!process.env.TRAVIS) {
```

Comment:
u can if you want `if(!process.env.TRAVIS || !process.env.TRAVIS_TAG) {return false}`